game_eggs/minecraft/java/folia/egg-folia.json

```diff
@@ -0,0 +1,77 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-07-04T01:55:37+02:00",
```

Comment:
Import this egg in to a up to date panel and export it again and fully replace it here.